src/errors/RequestResponseError.ts

```diff
@@ -2,16 +2,14 @@
  *  @module Errors
  */
 
-import { isObject } from "lodash";
-
 import { ErrorResponse } from "../api/RestAPI";
 
 type ErrorRequest = Omit<ErrorResponse, "status">;
 
 const serializeErrorResponse = ({ code, httpCode, errors }: ErrorRequest): string =>
     `Code: ${code}, HttpCode: ${httpCode}, Errors: ${(errors || [])
         .map((error) =>
-            isObject(error) ? ("field" in error ? `${error.reason} (${error.field})` : error.reason) : error
```

Comment:
Maybe it's better to reverse the line here?
```
typeof error === "string" ? error : ("field" in error ? `${error.reason} (${error.field})` : error.reason)
```

Reply:
Hum... That sounds reasonable to prevent the `undefined` and `null` case. Let me make the checks a bit more solid.

Reply:
if it's a number or array, then `error.reason` will throw

Reply:
```javascript
> const foo = 10
undefined
> foo
10
> foo.reason
undefined
```

Reply:
ah sorry i didn't notice the `"field" in error` part, ignore my comment above 🙇

Reply:
yeah so maybe `["number", "string"].indexOf(typeof error) > -1` is the better check

Reply:
indeed, and i forgot `typeof null` is actually `object`...

Reply:
![5](https://user-images.githubusercontent.com/5774716/100590517-da159f00-3337-11eb-8121-bc9ce40a83e5.jpg)

Reply:
`null` and `undefined` should be filterd now so it wont be crashing in this case. Let me log the error for `[string, number, boolean]`. Thanks for noticing it. The types being wrong here we can expect anything... We may try to fix them at one point to know what to expect though.